api/translator/src/main/java/com/project/translator/adapter/out/persistence/MessagePersistenceAdapter.java

```diff
@@ -151,6 +151,11 @@ private void createMessageTranslation(MessageDetails messageDetails) {
         final var languageId = messageDetails.language();
         final var originalMessageId = messageDetails.original_message();
         final var originalMessage = getMessageEntity(originalMessageId);
+        final var originalMessLang = originalMessage.getLanguage();
+        if(!isLanguageEnglish(originalMessLang)) {
```

Comment:
Widzę, że identyczny if pojawia się wyżej w kodzie. Może warto to wydzielić do osobnej metody?